src/concrete_minkowski_difference.jl

```diff
@@ -0,0 +1,71 @@
+export minkowski_difference, pontryagin_difference
+
+"""
+    minkowski_difference(P::LazySet{N}, Q::LazySet{N}) where {N<:Real}
+
+Concrete Minkowski difference (geometric difference) for a pair of
+lazy sets.
+
+### Input
+
+- `P` -- polytopic set
+- `Q` -- compact convex set that is subtracted from `P`
+
+### Output
+
+An `HQolytope` that corresponds to the Minkowski difference of `P` minus `Q` if
+`P` is bounded, and an `HPolyhedron` if `P` is unbounded.
+
+### Notes
+
+This function requires that the list of constraints of the lazy sets `P` is
+available and that the set `Q` is bounded.
+
+### Algorithm
+
+This function implements [Theorem 2.2], that we state next.
+Suppose ``U`` is compact and convex and ``U ~ V ≠ Ø``.
```

Comment:
I find it easier if you adapt the variable names to the code, so `U → P` and `V → Q`.